ArduCopter/switches.cpp

```diff
@@ -594,6 +594,18 @@ void Copter::do_aux_switch_function(int8_t ch_function, uint8_t ch_flag)
                 break;
             }
             break;
+
+        case AUXSW_SAFE_RTL:
+            if (ch_flag == AUX_SWITCH_HIGH) {
+                // engage SafeRTL (if not possible we remain in current flight mode)
+                set_mode(SAFE_RTL, MODE_REASON_TX_COMMAND);
+            } else {
```

Comment:
I'm pretty sure the compiler will optimize it to be the same thing anyways. To me it seems more logical the way it is because the else statement implies that we are checking (actually assuming) that the aux switch is low, and then we want to check the control mode. Again, this was just copied from existing code.